plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -227,7 +230,7 @@ GSTexture* GSDeviceOGL::CreateSurface(int type, int w, int h, bool msaa, int fmt
 
 	// A wrapper to call GSTextureOGL, with the different kind of parameter
 	GSTextureOGL* t = NULL;
-	t = new GSTextureOGL(type, w, h, fmt, m_fbo_read);
+	t = new GSTextureOGL(type, w, h, fmt, m_fbo_read, m_mipmap > 1 || m_filter > 2);
 	if (t == NULL) {
```

Comment:
Hmm. Comment replies don't appear in the feed?

Anyway, I think nullptr can only be returned if `new(std::nothrow)` is used.